ttnn/cpp/ttnn/operations/eltwise/unary_backward/device/unary_backward_op.cpp

```diff
@@ -0,0 +1,136 @@
+    grad_tensor.emplace_back(result);
+    return grad_tensor;
+}
+
+std::vector<Tensor> _clamp_min_bw(
+    const Tensor& grad, const Tensor& input, float min, const MemoryConfig& output_mem_config) {
+    std::vector<Tensor> grad_tensor;
+    Tensor minT = gte_unary(input, min, output_mem_config);
+    Tensor result = ttnn::multiply(grad, minT, std::nullopt, output_mem_config);
+    grad_tensor.emplace_back(result);
+    return grad_tensor;
+}
+
+
+std::vector<Tensor> _clamp_bw(
+    const Tensor& grad, const Tensor& input, float min, float max, const MemoryConfig& output_mem_config) {
+    std::vector<Tensor> grad_tensor;
+    Tensor minT = gte_unary(input, min, output_mem_config);
+    Tensor maxT = lte_unary(input, max, output_mem_config);
+    Tensor result = ttnn::logical_and(minT, maxT, std::nullopt, output_mem_config);
+    result = ttnn::multiply(grad, result, std::nullopt, output_mem_config);
+    grad_tensor.emplace_back(result);
+    return grad_tensor;
+}
+
+std::vector<Tensor> _assign_bw(const Tensor& grad, const Tensor& input, const MemoryConfig& output_mem_config) {
+    std::vector<Tensor> grad_tensor;
+    grad_tensor.emplace_back(grad);
+    return grad_tensor;
+}
+
+std::vector<Tensor> _multigammaln_bw(const Tensor& grad, const Tensor& input, const MemoryConfig& output_mem_config) {
+    std::vector<Tensor> grad_tensor;
+    Tensor digamma_result = ttnn::multiply(grad, tt::tt_metal::digamma(input, output_mem_config), std::nullopt, output_mem_config);
+    Tensor digamma_result_2 = ttnn::multiply(
+        grad, tt::tt_metal::digamma(add_unary(-0.5, input, output_mem_config), output_mem_config), std::nullopt, output_mem_config);
+
+    Tensor grad_result = ttnn::add(digamma_result, digamma_result_2, std::nullopt, output_mem_config);
+
+    digamma_result = ttnn::multiply(
+        grad, tt::tt_metal::digamma(add_unary(-1.0, input, output_mem_config), output_mem_config), std::nullopt, output_mem_config);
+    grad_result = ttnn::add(grad_result, digamma_result, std::nullopt, output_mem_config);
+
+    digamma_result = ttnn::multiply(
+        grad, tt::tt_metal::digamma(add_unary(-1.5, input, output_mem_config), output_mem_config), std::nullopt, output_mem_config);
+    grad_result = ttnn::add(grad_result, digamma_result, std::nullopt, output_mem_config);
+
+    grad_tensor.emplace_back(grad_result);
+    return grad_tensor;
+}
+
+std::vector<Tensor> _add_bw(
+    const Tensor& grad, const Tensor& input, float alpha, const MemoryConfig& output_mem_config) {
+    std::vector<Tensor> grad_tensor;
+    grad_tensor.emplace_back(grad);
+    return grad_tensor;
+}
+
+std::vector<Tensor> _unary_comp_bw(const Tensor& grad, const Tensor& input, float other, const MemoryConfig& output_mem_config) {
+    std::vector<Tensor> grad_tensor;
+    Tensor zero_grad = tt::tt_metal::zeros_like(grad, output_mem_config);
+    grad_tensor.emplace_back(zero_grad);
+    return grad_tensor;
+}
```

Comment:
What is the point of this function? `input` and `other` are unused

Reply:
backward operation for comparison ops would return only zero tensors of input shape. But this function was meant for generalizing all the comparison ops results. We'll update the code accordingly